src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -80,10 +80,10 @@
 	public List<String> biomeDictionaryTypes;
 	public List<String> defaultFeatures;
 
-	public List<SpawnTree> spawnTrees;
+	public List<TreeSpawn> treeSpawns;
 	public List<SpawnEntry> spawnEntries;
 
-	public boolean customTree;
+	public int treeType;

```

Comment:
This should remain boolean, only change to int when calling generateBiomePreviewPicture in the way I described in the previous review.